dropwizard-configuration/src/main/java/io/dropwizard/configuration/ResourceConfigurationSourceProvider.java

```diff
@@ -6,6 +6,11 @@
 public class ResourceConfigurationSourceProvider implements ConfigurationSourceProvider {
     @Override
     public InputStream open(String path) throws IOException {
+        InputStream result = getResourceAsStream(path);
+        return result == null && path.startsWith("/") ? getResourceAsStream(path.substring(1)) : result;
```

Comment:
Regardless, I think an inline comment could help. The very fact that we're having this conversation indicates that clarity is warranted.